docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -0,0 +1,77 @@
+Cluster Troubleshooting
+=======================
+
+This page describes some basic issues we have faced while deploying and
+operating the cluster.
+
+1. MongoDB Restarts
+-------------------
+
+We define the following in the ``mongo-ss.yaml`` file:
+
+.. code:: bash
+
+    resources:
+      limits:
+        cpu: 200m
+        memory: 3.5G
+
+So, when the MongoDB cache occupies a memory greater than 3.5GB, it is
```

Comment:
Cut the "So," and change 3.5GB to 5GB

Reply:
Maybe comment that it used to happen with 3.5GB but maybe the problem is gone now that it's 5GB.